examples/encodeFrame.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+import canmatrix.formats
+import sys
+import optparse
+
+# command line options...
+usage = """
+%prog [options]  matrix
+
+matrixX can be any of *.dbc|*.dbf|*.kcd|*.arxml
+"""
+
+parser = optparse.OptionParser(usage=usage)
+parser.add_option(
+    "-f", "--frames",
+    dest="frames",
+    help="encode list of frames",
+    default="*")
+
+(cmdlineOptions, args) = parser.parse_args()
+
+if len(args) < 1:
+    parser.print_help()
+    sys.exit(1)
+
+# load matrix
+db = canmatrix.formats.loadp(args[0], flat_import=True)
+
+#get all frames which match the commandline
+frames = db.glob_frames(cmdlineOptions.frames)
+
+#helper to read physical value from user
+def read_signal_value_from_user(signal):
+    a = input("Enter Value for " + signal.name + " ")
+
+    if signal.is_float:
+        return float(a)
```

Comment:
Decimal(a)?

Reply:
well, don't know. But at last in this use case, if `signal.is_float` an `IEEE`-float is put on the bus, so I think `float()` is right than. 
But not sure either...